legal-api/src/legal_api/services/search_service.py

```diff
@@ -269,3 +269,30 @@ def get_search_filtered_filings_results(business_json,
                 draft_results.append(draft)
 
         return draft_results
+
+    @staticmethod
+    def get_affiliation_mapping_results():
```

Comment:
I don't think you want the entire database, you probably want some identifiers passed down from the resource